x-pack/plugins/cases/public/common/use_cases_toast.test.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { renderHook } from '@testing-library/react-hooks';
+import { useToasts } from '../common/lib/kibana';
+import { TestProviders } from '../common/mock';
+import { useCasesToast } from './use_cases_toast';
+import { mockCase } from '../containers/mock';
+
+jest.mock('../common/lib/kibana');
+
+const useToastsMock = useToasts as jest.Mock;
+
+describe('Use cases toast hook', () => {
```

Comment:
I am wondering if is possible to test the context of the tooltip by either not mocking `useToasts` or by testing the `CaseToastSuccessContent` direclty.

Reply:
I'm trying to figure how to test the mounted component, I do want to write a test for it. If I can't find a way I'll test the `CaseToastSuccessContent` component